Configuration/AlCa/python/autoAlca.py

```diff
@@ -1,87 +1,40 @@
-                  "AlCaPhiSym"     : "",
-                  "AlCaP0"         : "",
-                  # ---------------------------------------------------------------------------------------------------------------------------
-                  "HcalNZS"        : "HcalCalMinBias",
-                  # This is in the AlCaRecoMatrix, but no RelVals are produced
-                  # 'TestEnablesTracker'  : 'TkAlLAS'
-                  # 'TestEnablesEcalHcal' : 'HcalCalPedestal'
-                  "MET" : "HcalCalNoise",
-                  "SingleMu" : "MuAlCalIsolatedMu+MuAlOverlaps+TkAlMuonIsolated+MuAlZMuMu+HcalCalHO",
-                  "DoubleMu" : "MuAlCalIsolatedMu+MuAlOverlaps+TkAlZMuMu",
-                  "DoubleMuParked" : "MuAlCalIsolatedMu+MuAlOverlaps+TkAlZMuMu",
-                  "MuOniaParked" : "TkAlJpsiMuMu+TkAlUpsilonMuMu",
-                  "DoubleElectron" : "EcalCalZElectron+EcalUncalZElectron",
-                  "StreamExpress" : "SiStripCalZeroBias+TkAlMinBias+SiStripPCLHistos+SiStripCalMinBias+SiStripCalMinBiasAAG+Hotline+LumiPixelsMinBias+SiPixelCalZeroBias+SiPixelCalSingleMuon+PPSCalTrackBasedSel",
-                  "StreamExpressHI" : "SiStripCalZeroBias+TkAlMinBiasHI+SiStripPCLHistos+SiStripCalMinBias+SiStripCalMinBiasAAG+SiPixelCalZeroBias"
+AlCaRecoMatrix = {
+                  "ALCALumiPixelsCountsExpress" : "AlCaPCCRandom",
+                  "AlCaLumiPixelsCountsPrompt"  : "AlCaPCCZeroBias+RawPCCProducer",
+                  # These two (AlCaPhiSym, AlCaP0) cannot run on RAW, they are just meant to run on the dedicated AlcaRAW so they do not enter the allForPrompt list
+                  "AlCaPhiSym"                  : "",
+                  "AlCaP0"                      : "",
+                  "ALCAPPS"                     : "PPSCalMaxTracks", # Express producer
+                  "AlCaPPS"                     : "PPSCalMaxTracks", # Prompt  producer
+                  "Charmonium"                  : "TkAlJpsiMuMu",
+                  "Commissioning"               : "HcalCalIsoTrk+HcalCalIsolatedBunchSelector+TkAlMinBias+SiStripCalMinBias",
+                  "Cosmics"                     : "SiPixelCalCosmics+SiStripCalCosmics+TkAlCosmics0T+MuAlGlobalCosmics",
+                  "DoubleEG"                    : "EcalCalZElectron+EcalUncalZElectron+HcalCalIterativePhiSym+HcalCalIsoTrkProducerFilter",
+                  "DoubleMuon"                  : "TkAlZMuMu+TkAlDiMuonAndVertex+MuAlCalIsolatedMu",
+                  "DoubleMuParked"              : "MuAlCalIsolatedMu+MuAlOverlaps+TkAlZMuMu",
+                  "EGamma"                      : "EcalESAlign+EcalUncalWElectron+EcalUncalZElectron+HcalCalIsoTrkProducerFilter+HcalCalIterativePhiSym",
+                  "Express"                     : "SiStripCalZeroBias+TkAlMinBias+SiStripPCLHistos+SiStripCalMinBias+SiStripCalMinBiasAAG+Hotline+SiPixelCalZeroBias",
```

Comment:
Is this `Hotline` also planned?

Reply:
I didn't find what this `Hotline` is, so I left it there for the moment...

Reply:
Hi @cms-sw/xpog-l2 I understand that his `Hotline` is a bunch of skims. Is this still relevant for Run-3? Thanks!

Reply:
@sam7k9621, do you have some recollection of these skims?

Reply:
For the record (info has already been shared with @tvami privately):
   * Group responsible of the `HotLine`: https://twiki.cern.ch/twiki/bin/viewauth/CMS/DatasetDefinitionTeam
   * details on the wf: https://indico.cern.ch/event/377707/

My impression is that this has been fully replace by Scouting.

Reply:
> I'll remove the Hotline then

Just to be on the safe side, I think you'd better check with EXO / PPD

Reply:
IIUC from a private mail thread with @lowette and @zdemirag , EXO needs more time to assess if the `Hotline` is needed or not. So for the moment I will leave this untouched.

Reply:
Just want to point out that it was not produced in 2018:
```
 dasgoclient -query='dataset dataset=/*Express/Run*Hotline*/* status=*'
/StreamExpress/Run2015B-Hotline-Express-v1/ALCARECO
/StreamExpress/Run2015C-Hotline-Express-v1/ALCARECO
/StreamExpress/Run2015D-Hotline-Express-v3/ALCARECO
/StreamExpress/Run2015D-Hotline-Express-v4/ALCARECO
/StreamExpress/Run2015E-Hotline-Express-v1/ALCARECO
/StreamExpress/Run2016A-Hotline-Express-v1/ALCARECO
/StreamExpress/Run2016A-Hotline-Express-v2/ALCARECO
/StreamExpress/Run2016B-Hotline-Express-v1/ALCARECO
/StreamExpress/Run2016B-Hotline-Express-v2/ALCARECO
/StreamExpress/Run2016C-Hotline-Express-v2/ALCARECO
/StreamExpress/Run2016D-Hotline-Express-v2/ALCARECO
/StreamExpress/Run2016E-Hotline-Express-v2/ALCARECO
/StreamExpress/Run2016F-Hotline-Express-v1/ALCARECO
/StreamExpress/Run2016G-Hotline-Express-v1/ALCARECO
/StreamExpress/Run2016H-Hotline-Express-v1/ALCARECO
/StreamExpress/Run2016H-Hotline-Express-v2/ALCARECO
/StreamExpress/Run2017A-Hotline-Express-v1/ALCARECO
/StreamExpress/Run2017A-Hotline-Express-v2/ALCARECO
/StreamExpress/Run2017A-Hotline-Express-v3/ALCARECO
/StreamExpress/Run2017B-Hotline-Express-v1/ALCARECO
/StreamExpress/Run2017B-Hotline-Express-v2/ALCARECO
/StreamExpress/Run2017C-Hotline-Express-v1/ALCARECO
/StreamExpress/Run2017C-Hotline-Express-v2/ALCARECO
/StreamExpress/Run2017C-Hotline-Express-v3/ALCARECO
/StreamExpress/Run2017D-Hotline-Express-v1/ALCARECO
/StreamExpress/Run2017E-Hotline-Express-v1/ALCARECO
/StreamExpress/Run2017F-Hotline-Express-v1/ALCARECO
```